open_humans/templates/pages/public-data-api.html

```diff
@@ -15,162 +15,443 @@
+            <p>
+              <a href="/api/public/datatypes/?source_project_id=176">
+              <code>/api/public/datatypes/?source_project_id=176</code>
+              </a>
+            </p>
+          </li>
+        </ul>
+      </li>
 
       <li>
-        <a href="{% domain %}/api/public-data/members-by-source/">
-          <code>{% domain %}/api/public-data/members-by-source/</code></a></li>
-    </ul>
+        <p>
+        <h4>DataType</h4>
+        Information about an individual DataType, specified by ID.
+        </p>
+        <p>
+          <b>URL path:</b>
+            <code>/api/public/datatypes/{id}/</code>
+        </p>
+        <p>
+          <b>Examples:</b>
+        </p>
+        <ul>
+          <li>
+            <p>
+              <a href="/api/public/datatype/4/">
+              <code>/api/public/datatype/4/</code>
+              </a>
+            </p>
+          </li>
+        </ul>
+      </li>
 
-    <h3>Query for a specific data type</h3>
+    </ul>
 
+    <hr>
+    <h3>Legacy endpoints</h3>
     <p>
-      Create a search URL for a specific data type:
+      These are deprecated.
     </p>
+    <ul>
+      <li>
+        <p>
+          <h4>Public data files</h4>
+        </p>
+        <p>
+          <b>URL path:</b> <a
+            href="/api/public-data/">
+            <code>/api/public-data/</code></a>
+        </p>
+        <p>
+          <b>Filter options:</b>
+        </p>
+        <ul>
+          <li><code>created_start</code> the start date to filter from
+            (inclusive)</li>
+          <li><code>created_end</code> the end date to filter to (inclusive)</li>
+          <li><code>source</code> the name of a data souce, like
+            <code>direct-sharing-134</code> or <code>direct-sharing-139</code>, or
+            a comma-separated list of sources, like
+            <code>direct-sharing-134,direct-sharing-139</code>.</li>
+          <li><code>username</code> a username, like <code>madprime</code></li>
+          <li><code>member_id</code> an Open Humans member ID, like
+            <code>94621811</code></li>
+        </ul>
+        <p>
+          <b>Examples:</b>
+          <p>
+            This query searches for the 5 most recently updated data files from user
+            <code>madprime</code>:
+          </p>
 
-    <p>
-      <select class="form-control input-lg select2-single" id="source-search">
-        <option></option>
-
-        {% for project in projects %}
-          <option value="{{ project.id_label }}">
-            {{ project.name }}
-          </option>
-        {% endfor %}
-      </select>
-    </p>
+          <p>
+            <code><a
+              href="/api/public-data/?username=madprime&limit=5">
+                /api/public-data/?username=madprime&limit=5</a>
+            </code>
+          </p>
 
-    <p>
-      <strong>URL:</strong> <span id="source-url"></span>
-    </p>
+          <p>
+            This query shows all Harvard Personal Genomes Project data updated on
```

Comment:
Should we maybe use another example? The PGP isn’t listed publicly right now as it’s inactive, isn’t it? Makes it harder for people to understand what’s going on?